pkgs/development/tools/parsing/tree-sitter/grammars/grammar-sources.nix

```
@@ -669,10 +669,10 @@
     hash = "sha256-2h1NPQtutTmdVKjydq/ZRvBCJ3YEiT+ZVcL72fb2m9M=";
   };
 
-  tera = {
-    version = "0-unstable-2025-07-22";
-    url = "github:uncenter/tree-sitter-tera/692937d52c8dfd91ce0dde722b9b4febbc9bc712";
-    hash = "sha256-3ZUark8lSH8mJwuLcvJPiQnE7+rlXstKkTPjvHzsE38=";
+  tera = rec {
+    version = "0.1.0";
+    url = "github:uncenter/tree-sitter-tera/v${version}";
```

```suggestion
  tera = {
    version = "0.1.0";
    url = "github:uncenter/tree-sitter-tera";
```

This can work without the recursive attribute. That tag will be auto-referenced based on the version by default: https://github.com/NixOS/nixpkgs/blob/1304e49fe377997c320e044a5142ac467988c6a9/pkgs/development/tools/parsing/tree-sitter/grammars/default.nix#L85

The approach above is only needed of repositories that don't use `vX.Y.Z` style release tags.